critiquebrainz/frontend/views/test/test_review.py

```diff
@@ -1,8 +1,29 @@
+from unittest.mock import MagicMock
 from critiquebrainz.frontend.testing import FrontendTestCase
 import critiquebrainz.db.review as db_review
 from critiquebrainz.db.user import User
 import critiquebrainz.db.users as db_users
 import critiquebrainz.db.license as db_license
+import critiquebrainz.frontend.external.musicbrainz_db.release as mb_release
+from flask import current_app
+
+
+def mock_get_entity_by_id(id, type='release_group'): # pylint: disable=unused-argument
```

Comment:
Why is `id` unused?

Reply:
`id` was unused as all tests in `test_review.py` use same entity ID for each type. I have fixed this as there might be other entities which may be tested in future. Thanks for pointing it out.